src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,54 @@
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  border: 1px dotted #000;
+  border-radius: 50%;
+}
+
+.watch__axis {
+  z-index: 2;
+  margin: 45px;
+  height: 10px;
+  width: 10px;
+  background-color: #ffa500;
+  border-radius: 50%;
+}
+
+.watch__hand {
+  position: absolute;
+  bottom: 50%;
+
+  transition: transform 1s;
+
+  &--short {
+    width: 4px;
+    height: 20px;
+    background-color: green;
+
+    transform-origin: 2px 20px;
+    animation: rotate 60s steps(60, end) 1;
+  }
+
+  &--long {
+    z-index: 1;
+    width: 2px;
+    height: 45px;
+    background-color: blue;
+
+    transform-origin: 1px 45px;
+    animation: rotate 5s steps(60, end) 12;
+  }
+}
```

Comment:
use nested SCSS features as in the page block

Reply:
Oh, sorry) Ok!